merlin/models/config/schema.py

```diff
@@ -52,10 +52,10 @@ def check_schema(self, schema=None):
         if self.REQUIRES_SCHEMA and not getattr(self, "_schema", None) and not schema:
             raise ValueError(f"{self.__class__.__name__} requires a schema.")
 
-    def __call__(self, *args, **kwargs):
-        self.check_schema()
-
-        return super().__call__(*args, **kwargs)
+    # def __call__(self, *args, **kwargs):
```

Comment:
If this is not strictly required, can we remove it. or was this an unintended comment?

Reply:
Yeah, you’re right. I will put it back